mgmt_cli_test.py

```diff
@@ -1147,3 +1147,33 @@ def test_suspend_and_resume_without_starting_tasks(self):
                 f'Task {current_task_status} did not remain in "{TaskStatus.STOPPED}" status, but instead ' \
                 f'reached "{current_task_status}" status'
         self.log.info('finishing test_suspend_and_resume_without_starting_tasks')
+
+    def test_manager_installed_and_functional(self):
+        """Verify that the Manager is installed and functional.
+
+        The test is intended for execution on non-main OS distributions (for example, debian10)
+        where the main goal is to execute installation test.
+        The rest of the checks are executed on Ubuntu distribution in test_manager_sanity.
+        """
+        self.log.info('starting test_manager_installed_and_functional')
+
+        manager_node = self.monitors.nodes[0]
+        scylla_node = self.db_cluster.nodes[0]
+        manager_tool = mgmt.get_scylla_manager_tool(manager_node=manager_node)
+
+        # Check scylla-manager and scylla-manager-agent are up and running (/ping endpoint)
+        manager_node.is_manager_server_up()
+        scylla_node.is_manager_agent_up()
+
+        # Check the sctool version method is callable
+        _ = manager_tool.sctool.version
```

Comment:
I wonder if it is better to put sctool version into logger (something like that)
```python
self.log.info("Got sctool version: %s", manager_tool.sctool.version)
```
or into `assert`
```python
assert manager_tool.sctool.version
```
instead of this assignment.